src/components/pattern-form/pattern-form.styles.js

```diff
@@ -1,68 +1,39 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { formStyles } from '../../configs/styles';
 
-export const useStyles = makeStyles((theme) => ({
-  patternItemUpdate: {
-    display: 'flex',
-    flexDirection: 'column',
-    margin: '20px 0'
-  },
-  patternDetails: {
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'space-around'
-  },
-  saveButton: {
-    margin: theme.spacing(2)
-  },
-  returnButton: {
-    margin: theme.spacing(2),
-    marginRight: 0
-  },
-  textField: {
-    margin: '10px 5px'
-  },
-  tabs: {
-    backgroundColor: 'white',
-    '& span.MuiTab-wrapper': {
-      color: '#3F51B5'
+export const useStyles = makeStyles((theme) => {
+  const {
+    DETAILS_STYLES,
+    INPUT_ERROR_STYLES,
+    IMAGE_NAME_STYLES,
+    SAVE_BUTTON_STYLES,
+    IMAGE_UPLOAD_CONTAINER_STYLES,
```

Comment:
All objects with styles are lowercased, why do we need this uppercase?

Reply:
Fixed